.prettierrc

```diff
@@ -0,0 +1,8 @@
+{
+  "semi": true,
+  "tabWidth": 4,
+  "useTabs": false,
+  "printWidth": 100,
```

Comment:
What this option means? The maximum size for each line?

Reply:
It's a way to say to Prettier roughly how long you’d like lines to be. Prettier will make both shorter and longer lines, but generally strive to meet the specified printWidth.